Dockerfile

```diff
@@ -23,12 +23,18 @@ ENV SRC_PATH="/bubuku"
 ADD ./bubuku "${SRC_PATH}/bubuku"
 ADD ./requirements.txt "${SRC_PATH}/"
 ADD ./setup.py "${SRC_PATH}/"
+
 RUN mkdir -p $KAFKA_LOGS_DIR/ && \
     cd "${SRC_PATH}" && \
     pip3 install --no-cache-dir -r "requirements.txt" && \
     python3 setup.py develop && \
     chmod -R 777 $KAFKA_LOGS_DIR && \
-    chmod 777 ${KAFKA_SETTINGS}
+    chmod 777 ${KAFKA_SETTINGS} && \
+    \
+    mkdir ${KAFKA_DIR}/logs && \
+    chmod 777 ${KAFKA_DIR}/logs
```

Comment:
I'm not fond of `777`, but seems to be the common style here ;)

Reply:
🤣